src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6999,6 +6999,22 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 tmp = ASR::make_LogicalConstant_t(al, loc, is_ascii,
                         ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
                 return;
+            } else if(attr_name == "isspace") {
+                /*
+                  * Specification:
+                  Return true if all characters in the input string are considered whitespace characters,
+                  as defined by the std::isspace function. Return false otherwise.
```

Comment:
```suggestion
                  as defined by CPython. Return false otherwise. For now we use the std::isspace function, but if we later discover that it differs from CPython, we will have to use something else.
```

Reply:
sure, I will do the analysis. but as every language tries to implement the same spec I used it here.